src/main/java/com/google/flourbot/entity/action/sheet/SheetReadRowAction.java

```diff
@@ -6,15 +6,15 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public final class SheetAppendAction extends Action {
+public final class SheetReadRowAction extends Action {
```

Comment:
The fact that it is so easy to change this method shows that it would be very useful to have parent SheetAction that gathers the common methods for all Sheet actions - don't worry about it for now though.